lib/Serialization/Deserialization.cpp

```diff
@@ -5306,7 +5306,7 @@ llvm::Error DeclDeserializer::deserializeDeclCommon() {
 
       // Do a quick check to see if this attribute is a move only attribute. If
       // so, emit a nice error if we don't have experimental move only enabled.
-      if (Attr->getKind() == DeclAttrKind::DAK_MoveOnly &&
+      if (Attr && Attr->getKind() == DeclAttrKind::DAK_MoveOnly &&
```

Comment:
Could also move this into `AddAttribute` where it's not null any more, or move into the if below (though I haven't looked where `skipAttr` is set, so maybe that's not possible?).

Reply:
It looks like `skipAttr` is set to true whenever `Attr` remains null. We could likely remove `skipAttr`, or just exit when we can't set the attribute.